### PR TITLE
[Longformer] Fix longformer documentation

### LINKED ISSUE #7015: Longformer global attention mask, 2 or 1?
This example in the LongFormer documentation (https://huggingface.co/transformers/model_doc/longformer.html )
```
attention_mask[:, [1, 4, 21,]] = 2  # Set global attention based on the task. For example,
...                                     # classification: the <s> token
...                                     # QA: question tokens
...                                     # LM: potentially on the beginning of sentences and paragraphs
```
seems to mislead the user to put a 2 as the global attention mask instead of a 1 this is described in the actual documentation. I'm guessing this is a mistake because (a) the example is directly copied from the AllenAI (https://github.com/allenai/longformer#how-to-use), and (b) the source code actually adds a +1 to the attention mask in the function `_merge_to_attention_mask` (https://huggingface.co/transformers/_modules/transformers/modeling_longformer.html#LongformerModel.forward). 

Could the documentation please be fixed or clarified? Thanks! 

Also what is the 1, 4, 21 in that example? 

### Who can help
 Longformer/Reformer: @patrickvonplaten

## Information

Model I am using (Bert, XLNet ...): LongFormer

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Fixes #7015

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Cool!
